src/components/Errors/Errors.stories.js

```diff
@@ -0,0 +1,49 @@
+import ErrorMessage from './ErrorMessage';
+
+const ALERT_MODIFIERS = ['info', 'warning', 'error', 'ok'];
```

Comment:
better to move this to (and export from) `ErrorMessage.js` so that you can use it in the proptypes:

```js
modifier: PropTypes.oneOf(ALERT_MODIFIERS),
```